package.json

```diff
@@ -213,6 +213,6 @@
     "node": ">=20"
   },
   "prisma": {
-    "seed": "node prisma/seed.js"
+    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
```

Comment:
`--compiler-options {\"module\":\"CommonJS\"}` looks a bit verbose but it seems to be the least invasive approach 🙂